ironic-config/httpd.conf.j2

```diff
@@ -1,6 +1,6 @@
 ServerRoot "/etc/httpd"
 {%- if env.LISTEN_ALL_INTERFACES | lower == "true" %}
-Listen [::]:{{ env.HTTP_PORT }}
+Listen {{ env.HTTP_PORT }}
```

Comment:
Is this still listening on all interfaces ?

Reply:
looks like from the Apache docs

Reply:
Yes, according to the apache documentation:
<img width="690" alt="image" src="https://github.com/user-attachments/assets/7b601d22-9369-43fc-9ae9-433bcea636b2">